container/resource/src/main/java/org/mobicents/slee/resource/ResourceAdaptorObjectImpl.java

```diff
@@ -22,7 +22,12 @@
 
 package org.mobicents.slee.resource;
 
-import java.io.Serializable;
+import org.apache.log4j.Logger;
+import org.mobicents.slee.container.resource.GracefullyStopableResourceAdaptor;
```

Comment:
is this new interface part of this PR? maybe you missed to git add the new file...